src/internal/HtmlColor.h

```diff
@@ -101,5 +132,11 @@ struct HtmlColor
     // 0x0000ff is blue
     // ------------------------------------------------------------------------
     uint32_t Color;
+
+    // ------------------------------------------------------------------------
+    // Array with all color names and its corresponding color codes
+    // The array ends with a NULL color name.
+    // ------------------------------------------------------------------------
+    static const HtmlColorName ColorNames[] PROGMEM;
```

Comment:
Ah, as an API writer who has to support it; I take the opposite track.  If there is no case to make it public; leave it private.  It is always easier to make something public than it is to hide once people start using it.  Reduce your API surface is always a good goal.